tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/io/UrlConnectionTestCase.java

```diff
@@ -0,0 +1,120 @@
+    private static final BackupAndRestoreAttributes backupMainServerGroup = new BackupAndRestoreAttributes.Builder(Address.of(SERVER_GROUP,"main-server-group")).build();
+    private static final BackupAndRestoreAttributes backupProfile = new BackupAndRestoreAttributes.Builder(Address.of(PROFILE, DEFAULT)).build();
+    private static final Administration administration = new Administration(client);
+
+    @BeforeClass
+    public static void setUp() throws IOException, TimeoutException, InterruptedException {
+        backupHost.backup();
+        backupOtherServerGroup.backup();
+        backupMainServerGroup.backup();
+        backupProfile.backup();
+
+        operations.batch(new Batch().remove(hostMaster.and("server-config","server-three"))); // /host=master/server-config=server-three:remove()
+        operations.batch(new Batch().remove(Address.of(SERVER_GROUP,"other-server-group")));                  // /server-group=other-server-group:remove()
+
+        operations.batch(new Batch().add(serverGroupMain, Values.of(PROFILE, DEFAULT)));
+        operations.batch(new Batch().writeAttribute(serverGroupMain,PROFILE, DEFAULT));
+
+        operations.batch(new Batch().writeAttribute(serverGroupMain,"socket-binding-group","standard-sockets"));
+
+        operations.batch(new Batch().remove(Address.of(PROFILE,DEFAULT).and("subsystem","undertow").and("server","default-server").and("http-listener","default")));
+        operations.batch(new Batch().writeAttribute(serverGroupMain.and("jvm",DEFAULT),"heap-size", "256m"));
+        operations.batch(new Batch().writeAttribute(serverGroupMain.and("jvm",DEFAULT),"max-heap-size", "256m"));
+
+        administration.reloadIfRequired();
+
+    }
+    @AfterClass
+    public static void tearDown()
+            throws IOException, InterruptedException, TimeoutException {
+        try {
+            backupHost.restore();
+            backupOtherServerGroup.restore();
+            backupMainServerGroup.restore();
+            backupProfile.restore();
+            administration.reloadIfRequired();
+        } finally {
+            client.close();
+        }
+    }
+
+
+    @Drone
+    private WebDriver browser;
+
+    @Inject
+    private Console console;
+
+    @Test
+    public void testUrlHttps() throws IOException {
+        FinderFragment fragment = getMainAttributesFinder();
+        fragment.column(Ids.SERVER).selectItem(Ids.build(ConfigUtils.getDefaultHost(), ConfigUtils.getDefaultServer()));
+        ServerPreviewFragment serverPreviewFragment = fragment.preview(ServerPreviewFragment.class);
+        if (!serverPreviewFragment.getUrlAttributeItem().getValueElement().getText().contains("https")) {
+            Assert.fail("URL should contain https! See WFLY-12115");
+        }
+    }
+
+    private FinderFragment getMainAttributesFinder() {
+        FinderFragment fragment;
+        if (ConfigUtils.isDomain()) {
```

Comment:
Please remove this, as the `@Category` annotation used on this test case makes it clear, that it is run in domain mode only.